src/main/java/org/jabref/logic/layout/format/ShortMonthFormatter.java

```diff
@@ -0,0 +1,20 @@
+package org.jabref.logic.layout.format;
+
+import java.util.Optional;
+
+import org.jabref.logic.layout.ParamLayoutFormatter;
+import org.jabref.model.entry.Month;
+
+public class ShortMonthFormatter implements ParamLayoutFormatter {
```

Comment:
```suggestion
public class ShortMonthFormatter implements LayoutFormatter {
```
If you do not need the setArgument method then you can use the LayoutFormatter interface